apps/deploy-web/src/components/new-deployment/BidRow.tsx

```diff
@@ -21,6 +21,7 @@ import { PriceEstimateTooltip } from "../shared/PriceEstimateTooltip";
 import { PricePerMonth } from "../shared/PricePerMonth";
 
 type Props = {
+  index: number;
```

Comment:
what would you say about this?
```suggestion
  testIndex: number;
```

Reply:
Sure